src/normalize.js

```diff
@@ -21,7 +21,12 @@ const extractFields = async (
     } else {
       // image fields have a mime property among other
       // maybe should find a better test
-      if (field !== null && field.hasOwnProperty('mime')) {
+      if (
+        field !== null &&
+        field.hasOwnProperty('mime') &&
+        field.hasOwnProperty('provider') &&
+        field.provider === 'local'
```

Comment:
The plugin should also load file from different providers. as it is a best practice with gatsby.